DakeAndDevileCorps/Screens/Search/VC/SearchViewController.swift

```diff
@@ -116,11 +116,11 @@ extension SearchViewController: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         if isShowingResult {
-            guard let cell = tableView.dequeueReusableCell(withIdentifier: "ResultTableViewCell", for: indexPath) as? ResultTableViewCell else { return UITableViewCell() }
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: "ResultTableViewCell", for: indexPath) as? ResultTableViewCell else { return UITableViewCell()}
```

Comment:
p1;

제가 넣어둔 NSObject+Extension에 있는 className를 사용하시면 "ResultTableViewCell"로 사용하지 않으셔도 됩니다!

```suggestion
            guard let cell = tableView.dequeueReusableCell(withIdentifier: ResultTableViewCell.className, for: indexPath) as? ResultTableViewCell else { return UITableViewCell() }
```
이런식으로 사용할 수 있어요!

Reply:
[56fba2f](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/37/commits/56fba2f414e4ce3a06e9ed63dbced472caa03e01)에서 반영했습니다!